shared/domains/chelonia/utils.js

```diff
@@ -278,18 +279,16 @@ export const keyAdditionProcessor = function (hash: string, keys: (GIKey | Encry
         key.meta.private.content &&
         !sbp('chelonia/haveSecretKey', key.id, !key.meta.private.transient)
       ) {
-        try {
-          decryptedKey = key.meta.private.content.valueOf()
-          decryptedKeys.push([key.id, decryptedKey])
-          storeSecretKey(key, decryptedKey)
-        } catch (e) {
-          console.warn(`Secret key decryption error '${e.message || e}':`, e)
-          // Ricardo feels this is an ambiguous situation, however if we rethrow it will
-          // render the contract unusable because it will undo all our changes to the state,
-          // and it's possible that an error here shouldn't necessarily break the entire
-          // contract. For example, in some situations we might read a contract as
-          // read-only and not have the key to write to it.
```

Comment:
Did this situation get cleared up or is this comment still needed?

Reply:
After refactoring, there isn't any exception to re-throw, so I don't think this is relevant.